### DISCUSSION TITLE
Cannot lock node positions

### DISCUSSION THREAD

**Discussion by Julianwoolf** [xyflow/xyflow#3457] [Q&A] [answered]
Hi,
I'm using custom nodes with a text input field, custom edges with a text input field and an "insert node" button, and the auto layout hook where nodesDraggable=false. However, when typing in the text fields and using arrow keys to move within the text also the node is moved in the respective direction (although drag is deactivated). Using tailwind, the className="nodrag" attribute seems to not be working. Only elementsSelectable=false has disabled this behavior so far, but then the textfield and insert node button on my edges are not accessible anymore.

Is there a way to disable the arrow key inputs completely?

I'm using reactflow v.11.0.

Thanks in advance.
Julian

**Comment by bcakmakoglu** [comment, marked as answer]
You probably want to use [`disablekeyboarda11y`](https://reactflow.dev/docs/api/react-flow-props/#disablekeyboarda11y) which is the prop that allows you to disable/enable moving nodes with the keyboard.

**Reply by Julianwoolf** [reply]
Perfect, that solved it! Can't believe I didn't see this before. Thanks a lot!
